espnet2/bin/whisper_export_vocabulary.py

```diff
@@ -54,9 +56,14 @@ def export_vocabulary(output: str, whisper_model: str, log_level: str):
     # NOTE (Shih-Lun): extra tokens (for timestamped ASR) not
     #                  stored in the wrapped tokenizer
     full_vocab_size = 51865 if whisper_model == "whisper_multilingual" else 51864
+
     for i in range(full_vocab_size - vocab_size):
         fout.write("()" + "\n")
 
+    if sot_asr:
```

Comment:
Is it possible to import custom tokens from a file, like `additional_vocab`. This would allow us to add any tokens as needed, rather than only adding a `<sc>` token for the SOT training.